src/qubes_miragevpn.md

```diff
@@ -0,0 +1,64 @@
+One advantage of MirageVPN is its ability to produce a [unikernel][mirageos],
+i.e. a fully-fledged operating system for a virtualizer such as [Xen]. In this
+respect, [QubesOS][qubesos] users can secure their VMs' Internet connections via
+a MirageVPN client.
+
+Unlike a standard OpenVPN client, the VM used is smaller and its attack surface
+just as reduced, as the operating system only does VPN: an OpenVPN client would
+require a kernel (like Linux), a kernel configuration to redirect VM connections
+to the VPN tunnel, and several libraries installed in order to function.
+
+In this chapter, we'll look at how to configure and install a MirageVPN client
+for QubesOS. We'll assume that the OpenVPN server is configured as described in
+[this handbook](./simple_openvpn_server.md).
+
+## Download and configuration
+
+You can download the unikernel from its official repository:
+https://github.com/robur-coop/qubes-miragevpn
+
+Next, copy the unikernel to dom0 with this command (from a dom0 terminal). In
+this example, we've uploaded the unikernel to the _app-vm_ `personal`:
+```sh
+$ mkdir -p /var/lib/qubes/vm-kernels/qubes-miragevpn/
+$ cd /var/lib/qubes/vm-kernels/qubes-miragevpn/
+$ qvm-run -p personal 'cat qubes-miragevpn.xen' > vmlinuz
+```
+
+Still from dom0, we now need to create a new VM from the downloaded image. An
+empty initramfs file must also be created:
```

Comment:
It's not anymore since recently with Qubes 4.2 :) https://github.com/QubesOS/qubes-doc/pull/1402